### PR TITLE
fix prefix even moar better

### PR DESCRIPTION
**Pull request by notwaldorf**
Fixes https://github.com/PolymerElements/paper-input/issues/228. Looking at the code some more, I think we don't need to listen to children at all, and just fix the floating label style to always align to the edge (which is a no-op if there's no prefix)

Bumped down the polymer dependency version as a result.

### CODE REVIEW

File: bower.json [hunk 1]
```diff
@@ -27,7 +27,7 @@
   "homepage": "https://github.com/PolymerElements/paper-input",
   "ignore": [],
   "dependencies": {
-    "polymer": "Polymer/polymer#^1.2.0",
```
Comment (cdata) [inline bower.json]: It's a little weird that a previous version of this element requires v1.2.0, but a more recent version requires v1.1.0.. but, I guess it's not any weirder than the rest of our dependency graph.

File: paper-input-container.html [hunk 2]
```diff
@@ -562,16 +547,15 @@
 
         if (alwaysFloatLabel || _inputHasContent) {
           cls += ' label-is-floating';
+          // If the label is floating, ignore any offsets that may have been
+          // applied from a prefix element.
+          this.$.labelAndInputContainer.style.position = 'static';
```
Comment (cdata) [inline paper-input-container.html]: :star:

**Comment by cdata** [comment]
I assume there are already tests that cover this positioning?

Otherwise, LGTM :+1:

File: bower.json (hunk 1 above)
Comment (notwaldorf) [inline bower.json]: I don't have to change it. I think I was being overly pedantic, maybe. Should I revert it?

**Comment by notwaldorf** [comment]
@cdata there are no tests that cover positioning for this label :(
I'll look into it.

File: bower.json (hunk 1 above)
Comment (cdata) [inline bower.json]: No, it's fine. It's just weird!!

**Comment by cdata** [comment]
Thanks for tests :100: LGTM :+1:

**Pull request merged**